vic/drivers/classic/src/read_vegparam.c

```diff
@@ -462,10 +461,10 @@ read_vegparam(FILE  *vegparam,
     }
 
     // Bare soil tile
```

Comment:
I'm confused.  Wouldn't we want to raise an error if Cv_sum is still < 1 at this point?  The comment on line 465 is not true in general.  I use a veg library that contains an explicit bare soil class.  And I thought we were going to require that the veg library contain an explicit soil class so that no more rescaling happens behind the curtain.  Again, tonic already figures out whether the veg library is missing a bare soil class and creates one if not (although tonic's logic in making that inference could probably stand to be improved).  If we write an ascii output function for tonic (say, netcdf to ascii conversion) then tonic could be used to upgrade ascii parameters as well.

Or, are we keeping the old behavior for VIC classic?  I still would not say that bare soil is not mentioned in the veg library...